components/forms/farmer-journal/ChickenFeeding.tsx

```diff
@@ -16,7 +16,7 @@ export default function ChickenFeeding() {
 
     <Label htmlFor="fieldGivenFeed" required>Total given</Label>
     <div className="join">
-      <NumberInput name="fieldGivenFeed" className="w-full join-item" required step={0.01} />
+      <NumberInput name="fieldGivenFeed" className="w-full join-item" step={0.01} />
```

Comment:
Not required?

Reply:
Yep, `react-hook-form` and `zod` will handle the validation